src/sentry/snuba/outcomes.py

```diff
@@ -281,6 +283,11 @@ def run_outcomes_query(query: QueryDefinition) -> Tuple[ResultSet, ResultSet]:
         referrer="outcomes.totals",
         limit=10000,
     )
+    result_totals = _format_rows(result["data"], query)
+    return result_totals
```

Comment:
```suggestion
    return _format_rows(result["data"], query)
```

You could fold these together 🏌️‍♂️